### DISCUSSION TITLE
Pipeline config application overriding

### DISCUSSION THREAD

**Discussion by ssrilakshmanan** [casid/jte#406] [Q&A] [answered]
Hi,

I am loving JTE. Thank you very much for creating this tool.

Not sure whether I am doing something wrong or I have misunderstood, either way please correct me.

I am using maven to build my application.

I have application_environments{  dev{ }  } block defined in the pipeline_config.groovy in the Governance Tier.
I want to merge/override some properties when the application builds, but would like the application to inject  properties from a config file.

For example having a pipeline_config.groovy in my application that will merge additional properties to the dev{} block.

I am able to retrieve custom properties defined in pipeline_config.groovy in the Governance Tier, but none from my application. Is it possible to inject additional properties from an application config file?

Any suggestion is greatly appreciated.

**Comment by marcospereira** [comment, marked as answer]
Wrong JTE. :-)

You want https://github.com/jenkinsci/templating-engine-plugin.
